core/dbt/task/base.py

```diff
@@ -64,13 +65,15 @@ def pre_init_hook(cls, args):
         """A hook called before the task is initialized."""
         if args.log_format == 'json':
             log_manager.format_json()
+            event_logger.format_json = True
         else:
             log_manager.format_text()
```

Comment:
it seems like these blocks won't need to exists once we refactor the early logs right? can we comment to that effect so the refactor goes smoother?